index.html

```diff
@@ -286,6 +286,42 @@ <h2 class="topic">
             </ul>
         </div>
     </section>
+
+    <footer id="contact" class="footer">
+        <div class="contact-me">
+            <h2>Contact me</h2>
```

Comment:
- [ ] Kindly correct the `font-weight` to be similar to the Figma template.

![image](https://user-images.githubusercontent.com/58642949/186796804-528ae123-1669-42c1-9f5a-f8c26660e75b.png)